src/components/booksWrapper.jsx

```diff
@@ -1,23 +1,46 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import Book from './book';
 import BookForm from './bookForm';
+import { fetchBooks } from '../redux/books/bookSlice';
 
 function BooksWrapper() {
-  const books = useSelector((data) => data.books);
+  const dispatch = useDispatch();
+
+  const { books, isLoading } = useSelector((data) => data.books);
+
+  // console.log(books)
+  // const {books} = useSelector(data=> data.books)
```

Comment:
[OPTIONAL] Consider removing unused code pieces from your codebase inorder to make them clean